language/move-lang/tests/move_check/locals/assign_partial_resource.exp

```diff
@@ -0,0 +1,66 @@
+error: 
+
+   ┌── tests/move_check/locals/assign_partial_resource.move:6:21 ───
+   │
+ 7 │         r = R{};
+   │         ^ Invalid assignment to local 'r'
+   ·
+ 6 │         if (cond) { r = R{}; };
+   │                     - The local might contain a resource value due to this assignment. The resource must be used before you assign this to this local again
```

Comment:
I don't like one part of the message: "assign this to this local". Too many "this"s. Can you drop the first one and say "assign to this local"?